src/components/home/PanelResult.vue

```diff
@@ -18,43 +18,87 @@
       @closing="showDialog = false"
     >
       <template v-slot:title>
-        <span v-if="geneType === 'genesInPanel'">
-          {{ $t('panel-result.dialog.title-genes-in-panel') }}:
-          {{ panelName }}
-        </span>
-        <span v-else>
-          {{ $t('panel-result.dialog.title-genes-not-in-panel') }}:
-          {{ panelName }}
-        </span>
+        {{ $t('panel-result.dialog.title') }}:
+        {{ panelName }}
       </template>
       <template v-slot:content>
-        <v-virtual-scroll :items="genes" height="400" item-height="64">
-          <template v-slot:default="{ item }">
-            <v-list-item :key="item">
-              <v-list-item-content>
-                <v-list-item-title>
-                  <v-chip
-                    :outlined="chipOutlined"
-                    class="ma-1"
-                    :color="formatState(item)"
+        <v-row no-gutters>
```

Comment:
Keep the gutters or add some padding/margin between the v-col. It's weird when the left one doesn't have a scroll bar and the right column does. Or force the scrollbar even if not needed.
Also, padding needed on top and less paddings left and right. It could be an issue inside the DialogTemplate.

Reply:
updated